apps/glusterfs/pendingop_entry.go

```diff
@@ -380,3 +381,49 @@ func PendingOperationEntrySelection(
 	}
 	return selection, nil
 }
+
+func (p *PendingOperationEntry) consistencyCheck(db Db) (consistent bool, inconsistencies []string) {
+
+	consistent = true
+
+	for _, action := range p.Actions {
+		switch action.Change {
+
+		case 1:
+			fallthrough
+		case 3:
+			if p.Id != db.Bricks[action.Id].Pending.Id {
+				inconsistencies = append(inconsistencies, fmt.Sprintf("pending op %v id in change missing %v not found in bricks", p.Id, action.Id))
+				consistent = false
+			}
+		case 2:
+			fallthrough
+		case 4:
+			fallthrough
+		case 5:
+			fallthrough
+		case 9:
+			fallthrough
+		case 10:
+			fallthrough
```

Comment:
can't we write this as `case 2,4,5,...:` without fallthrough?
In addition I'd really rather we use the type names and not the raw values. So `OpAddBrick` rather than 1.

Reply:
done